### PR TITLE
I made a grammar test for English lesson1

### PR DESCRIPTION
**Pull request by DenisAbramovDm**

### CODE REVIEW

File: homework/english/denis_abramov/lecture1/grammar.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Упражнения Present Simple и Present Continuous
+## Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+Is your mum working today?
+Why this boys are talking now?
+Where he lives?
+Is their mother cookes very well?
```
Comment (Alexandr-R-94) [inline homework/english/denis_abramov/lecture1/grammar.md:7]: It seems to me that it is worth using a different time here.

File: homework/english/denis_abramov/lecture1/grammar.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Ann usually makes doll dresses herself.
+Is his wife having coffee at the moment?
+Let’s go for a walk. It is not raining.
+Look! The boys are coming home.
+Does Maila like Chinese food?
+My grandmother doesn't work. She is on pension.
+What is your brother watching now?
+Where do you live?
+Do You play the piano well?
+Where their brother is swimming now?
```
Comment (Alexandr-R-94) [inline homework/english/denis_abramov/lecture1/grammar.md:21]: the situation is similar to line 5

**Review by Alexandr-R-94 — COMMENTED** [review]
Check out these suggestions. I am not 100% sure that my opinion is correct

**Review by Alexandr-R-94 — APPROVED** [review]

File: homework/english/denis_abramov/lecture1/grammar.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Упражнения Present Simple и Present Continuous
+## Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+Is your mum working today?
+Why this boys are talking now?
+Where he lives?
```
Comment (olgaz-skiba) [inline homework/english/denis_abramov/lecture1/grammar.md:6]: Where does he live?

File: homework/english/denis_abramov/lecture1/grammar.md (hunk 1 above)
Comment (olgaz-skiba) [inline homework/english/denis_abramov/lecture1/grammar.md:7]: Does their mother cook very well?

**Review by olgaz-skiba — APPROVED** [review]
Igfm

**Pull request merged**